x-pack/test/functional/apps/dashboard/reporting/screenshots.ts

```diff
@@ -122,14 +122,32 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         const reportData = await PageObjects.reporting.getRawPdfReportData(url);
         const reportFileName = 'dashboard_preserve_layout';
         const sessionReportPath = await writeSessionReport(reportFileName, reportData, 'png');
-        const percentSimilar = await checkIfPngsMatch(
+        const percentDiff = await checkIfPngsMatch(
           sessionReportPath,
           getBaselineReportPath(reportFileName, 'png'),
           config.get('screenshots.directory'),
           log
         );
 
-        expect(percentSimilar).to.be.lessThan(0.1);
+        expect(percentDiff).to.be.lessThan(0.09);
```

Comment:
with value being (0.08) :  I hit this error 
``` Error: expected 0.08489858128285613 to be below 0.08
       at Assertion.assert (/Users/rashmikulkarni/Documents/kibana/packages/kbn-expect/expect.js:100:11)
       at Assertion.lessThan.Assertion.below (/Users/rashmikulkarni/Documents/kibana/packages/kbn-expect/expect.js:336:8)
       at Function.lessThan (/Users/rashmikulkarni/Documents/kibana/packages/kbn-expect/expect.js:531:15)
       at Context.<anonymous> (test/functional/apps/dashboard/reporting/screenshots.ts:132:35)
       at Object.apply (/Users/rashmikulkarni/Documents/kibana/packages/kbn-test/src/functional_test_runner/lib/mocha/wrap_function.js:84:16)
```
changing it to (0.09) - solves it.